.eslintrc

```diff
@@ -9,10 +13,24 @@
   ],
   "plugins": [
     "flowtype",
-    "prettier"
+    "prettier",
+    "jest"
   ],
   "rules": {
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "bracketSpacing": true,
+        "printWidth": 80,
+        "singleQuote": false,
+        "tabWidth": 2,
+        "trailingComma": "all"
+      }
+    ],
+    "import/no-extraneous-dependencies": "off",
```

Comment:
nope

Reply:
I agree with @oreqizer . If this is causing issues in tests then use a glob pattern to fix it. https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md

Reply:
It's not working for yarn workspaces. Globs didn't helped for all cases.
https://github.com/benmosher/eslint-plugin-import/issues/458#issuecomment-361352578

Reply:
so turn off where needed